__main__.py

```diff
@@ -10,7 +10,7 @@
     if not schema.is_file() or not data.is_file():
         print("Missing schema or data file!")
     else:
-        name = input("Enter a table name: ")
+        name = raw_input("Enter a table name: ")
```

Comment:
Was this working for you locally?

Reply:
If you pull this branch down to your Windows machine, does _this branch_ work?

Reply:
Just "input" works for me locally. What version of Python are you using? It says in documentation for Python 3, the version I was using for this project, that "raw_input" is not valid anymore and has been renamed to just "input"!